### PR TITLE
Number of contributors and size of a Github repository

### PR DESCRIPTION
**Pull request by Namyalg**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #216 

### Type of change

**Select all that applies**:
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by tejan-singh** [comment]
I am not able to get outputs. Can you please check running the script again?

**Comment by Namyalg** [comment]
I have tested the script on multiple repos, it works fine

**Comment by SANKET7738** [comment]
Please squash the commits @Namyalg

**Comment by Namyalg** [comment]
I have done it now

File: Python/Github-repo-and-contributors/get_info.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import requests
+import json
+
```
Comment (yashaswibiyahut) [inline Python/Github-repo-and-contributors/get_info.py:4]: Add proper comments

**Comment by yashaswibiyahut** [comment]
Please squash the commits still there are 7 commits

**Comment by Namyalg** [comment]
I need some help with squashing the commits, how exactly should I go about?

**Pull request merged**